ansible/mainnet.yml

```
@@ -16,7 +16,7 @@
   vars_files: layout/mainnet.yml
   vars:
     # WARNING: Since these are Eth 2 bootnodes we need to keep the keys and IPs unchanged.
-    beacon_node_netkey: '{{lookup("bitwarden", "nimbus/netkey", field=hostname)}}'
+    beacon_node_netkey: '{{lookup("vault", "netkey", field=hostname, stage="beacon-node")}}'
```

Why override the stage and not keep the default one `mainnet` ?

---

Yeah, the service is called `beacon-node`, so it should be more like:
```suggestion
    beacon_node_netkey: '{{lookup("vault", "beacon-node/netkey", field=hostname)}}'
```

---

Right! I will remove the stage override.
`mainnet` makes more sense here.